openssl-dynamic/src/main/java/org/apache/tomcat/jni/CertificateVerifier.java

```diff
@@ -20,6 +20,73 @@
  */
 public interface CertificateVerifier {
 
+    int X509_V_OK = 0;
```

Comment:
would be nice to have a link as to where these were derived from ... I suspect some openssl header file though :wink: